int32 x
int32 y
int32 disparity
float32 fx
float32 fy
float32 px
float32 py
float32 a
float32 b
---
float32 X
float32 Y
float32 Z
